### PR TITLE
Add documentation for unknown errors

### PR DESCRIPTION
**Pull request by Princesseuh**
## Changes

We still generate docs links for unknown errors, so we should have documentation for them!

## Testing

N/A

## Docs

It's docs all the way down!

/cc @withastro/maintainers-docs for feedback!

### CODE REVIEW

File: packages/astro/src/core/errors/errors-data.ts [hunk 1]
```diff
@@ -590,6 +633,13 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 	 * @name CLI Errors
 	 */
 	// CLI Errors - 8xxx
+	/**
+	 * @docs
+	 * @description
+	 * Astro encountered an unknown error while starting one of its CLI command. The error message should contain more information.
```
Comment (Yan-Thomas) [inline packages/astro/src/core/errors/errors-data.ts:639]: I believe that when we use "one of" we need to use a plural noun instead.
```suggestion
	 * Astro encountered an unknown error while starting one of its CLI commands. The error message should contain more information.
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks @Princesseuh, great work tackling this. I have a small suggestion and then LGTM!

File: packages/astro/src/core/errors/errors-data.ts [hunk 2]
```diff
@@ -676,7 +734,7 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 		hint: 'See https://docs.astro.build/en/guides/content-collections/ for more on the `slug` field.',
 	},
 
-	// Generic catch-all
+	// Generic catch-all - Only use this in extreme cases, like if there was a cosmic ray bit flip
```
Comment (natemoo-re) [inline packages/astro/src/core/errors/errors-data.ts:737]: 😆 Nice clarification

**Review by natemoo-re — APPROVED** [review]
Looks great, nice job!

**Review by Yan-Thomas — APPROVED** [review]

**Pull request merged**